conf/core/ConfigOptions.asciidoc

```diff
@@ -3860,6 +3860,37 @@ Add a tag with the bounding box for each element
 Overpass API URL host name. Used by the JSON readers to distinguish Overpass JSON web sources from
 GeoJSON web sources.
 
+=== pbf.writer.compression.level
+
+* Data Type: int
+* Default Value: `-1`
+
+The compression level to use when writing an OSM PBF. Valid values are -1 to 9. -1 is "default" or equivalent to about 7.
+0 is uncompressed. 1 is fastest. 9 is best compression.
+
+=== pbf.writer.flush.output
+
+* Data Type: bool
+* Default Value: `true`
+
+When `true`, the OSM PBF writer will regularly flush the output.
+
+=== pbf.writer.granularity
+
+* Data Type: int
+* Default Value: `100`
+
+The granularity of the written OSM PBF. The granuarlity is in units of nanodegrees. The larger the value the coarser the data
+stored, but potentially higher compression. The default of 100nm introduces about 8.5mm of error on average and 15.5mm of
+error maximum with 2000 random points.
```

Comment:
Changes look good but you haven't used this value in the PBF code.  Should be on line `84` of `OsmPbfWriter.cpp`

Reply:
Doh!
I thought I got all of them.  I was going to group them and then decided not to.